lemur/manage.py

```diff
@@ -583,24 +651,18 @@ def publish_verisign_units():
 
 
 def main():
-    manager.add_command("start", LemurServer())
-    manager.add_command("runserver", Server(host="127.0.0.1", threaded=True))
-    manager.add_command("clean", Clean())
-    manager.add_command("show_urls", ShowUrls())
```

Comment:
do we have any equivalent command, or did we need to remove entirely?

Reply:
see https://github.com/Netflix/lemur/pull/4468#discussion_r1235286626